### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,16 +87,17 @@ below.
...
-| spark-3.1-bigquery                    |         |         |         | &check; | &check; | &check;                 | &check;                 | &check;                 |
-| spark-2.4-bigquery                    |         | &check; | &check; |         |         |                         |                         |                         |
-| spark-bigquery-with-dependencies_2.13 |         |         |         |         |         |                         | &check;                 | &check;                 |
-| spark-bigquery-with-dependencies_2.12 |         |         | &check; | &check; | &check; | &check;                 |                         |                         |
-| spark-bigquery-with-dependencies_2.11 | &check; | &check; |         |         |         |                         |                         |                         |
+| Connector \ Dataproc Image            | 1.3     | 1.4     | 1.5     | 2.0     | 2.1     | 2.2     | Serverless<br>Image 1.0 | Serverless<br>Image 2.0 | Serverless<br>Image 2.1 |
```
Comment (vishalkarve15) [inline README.md:90]: Shouldn't we add serverless 2.2 as well?

**Comment by tom-s-powell** [comment]
Not sure if this is the best place to ask, but could I check on **Connector to Spark Compatibility Matrix**? It says that `spark-bigquery-with-dependencies_2.12` is compatible with Spark 3.4 and 3.5, but it seems that `spark-bigquery-pushdown` does not include versions for 3.4 and 3.5 and therefore not sure all functionality is supported? For example, `BigQueryConnectorUtils#disablePushdownSession` does not work on Spark 3.4 due to not being able to find a compatible version of `SparkBigQueryPushdown`.

**Comment by tom-s-powell** [comment]
I raised https://github.com/GoogleCloudDataproc/spark-bigquery-connector/pull/1168.

**Comment by davidrabinowitz** [comment]
obsolete

**Pull request closed without merging**